src-electron/db/query-endpoint.js

```diff
@@ -265,6 +265,40 @@ ORDER BY C.CODE
   })
 }
 
+/**
+ * Retrieves endpoint cluster events.
+ *
+ * @param {*} db
+ * @param {*} clusterId
+ * @param {*} endpointTypeId
+ * @returns promise that resolves into endpoint cluster events
+ */
+async function selectEndpointClusterEvents(db, clusterId, endpointTypeId) {
+  return dbApi.dbAll(
+    db,
+    `
+SELECT
+  E.EVENT_ID,
+  E.NAME,
+  E.CODE,
+  E.MANUFACTURER_CODE,
+  E.IS_OPTIONAL
```

Comment:
You might want to select more columns here; see the set of things that `dbMapping.map.event` maps.

Reply:
updated